lib/rgb_matrix/animation.ex

```diff
@@ -1,109 +1,88 @@
 defmodule RGBMatrix.Animation do
-  @moduledoc """
-  Provides a data structure and functions to define an RGBMatrix animation.
+  alias Layout.LED
 
-  There are currently two distinct ways to define an animation.
+  @callback new(leds :: list(LED.t()), config :: any) :: {render_in, any}
```

Comment:
`render_in` is a required return value from all animation callbacks, because you can't know what an animation designer wants to do.  I have several examples of animations that have to return a `render_in` from `new` and some that don't.  Removing this would be a re-design of the animation system.

Reply:
https://github.com/ElixirSeattle/xebow/issues/66